pxr/usd/usdGeom/schema.usda

```diff
@@ -2292,6 +2292,35 @@ class Camera "Camera" (
     However, it follows that if even one property is authored in the correct
     scene units, then they all must be.
 
+    \\section UsdGeom_CameraExposure Camera Exposure Model
+
+    UsdGeomCamera models exposure by a camera in terms of exposure time, ISO,
+    f-number, and exposure compensation, mirroring the controls on a real camera.
```

Comment:
From a high level view: if we don't author the f-number and exposure time, should they fallback to be the fStop and the shutter interval length (converted to seconds)? Or would that be undesirable?

Reply:
That would be undesirable as it would mean that when users tweak the depth of field or their motion blur settings, the image would change brightness, not to mentioned the backward-compatibility I mentioned in the other comment. 

You can imagine cases in which an application wanting to model a real camera might want these properties linked to "make it more real", but IMO that is a UX thing, not a schema thing.

Reply:
Makes sense.